clang/test/3C/multidef2_xfail.c

```diff
@@ -0,0 +1,15 @@
+// RUN: rm -rf %t*
```

Comment:
I waited until after your merge so that your new knowledge of proper testing could be in the codebase when I built new tests out of old ones. Your improvements didn't get committed? I'll see what I can do to recreate your work.

Reply:
Right, I didn't think it was appropriate to include my change to the declaration merging tests in #412 because it wasn't really related to the rest of #412.  I could have submitted it as a separate PR.  I thought it might be less overhead if you cherry-picked it into this PR: it's the sort of cleanup of existing tests that we would generally consider acceptable to do in the same PR as adding new similar tests, though typically the same author would do both.  But I'm happy to submit the change as a separate PR now if you prefer.

Reply:
Used some of the given code, changed names. There are already multiple merges across files in tests.